zz.home-work/src/components/todo-form.tsx

```diff
@@ -35,8 +51,13 @@ export const TodoForm = ({
         variant="outlined"
         fullWidth
         value={todo}
-        onChange={(e) => updateTodo(e.target.value)}
+        onChange={(e) => {
+          if (e.target.value.length <= 99) {
+            updateTodo(e.target.value);
+          }
+        }}
         style={{ marginBottom: '1rem' }}
+        slotProps={{ htmlInput: { maxLength: 100 } }}
```

Comment:
p1;
100이나 99라는 상수값을 같은맥락인데 굳이 2가지로 나누지 않으셔도 좋을 듯 합니다.
100을 고정값으로 둔 후 변수로 활용하시는 것을 추천드립니다.

```suggestion
const MAX_LENGTH = 100;
          if (e.target.value.length < MAX_LENGTH) {
            updateTodo(e.target.value);
          }
        }}
        style={{ marginBottom: '1rem' }}
        slotProps={{ htmlInput: { maxLength: MAX_LENGTH } }}
```